windows/CodePushPackage.cs

```diff
@@ -0,0 +1,364 @@
+﻿using Newtonsoft.Json;
+using Newtonsoft.Json.Linq;
+using System;
+using System.IO;
+using System.IO.Compression;
+using System.Threading;
+using System.Threading.Tasks;
+using Windows.Storage;
+using Windows.Storage.Streams;
+using Windows.Web.Http;
+
+namespace CodePush.ReactNative
+{
+    class CodePushPackage
+    {
+        private readonly string CODE_PUSH_FOLDER_PREFIX = "CodePush";
+        private readonly string CURRENT_PACKAGE_KEY = "currentPackage";
+        private readonly string DIFF_MANIFEST_FILE_NAME = "hotcodepush.json";
+        private readonly string DOWNLOAD_FILE_NAME = "download.zip";
+        private readonly string DOWNLOAD_URL_KEY = "downloadUrl";
+        private readonly string PACKAGE_FILE_NAME = "app.json";
+        private readonly string PACKAGE_HASH_KEY = "packageHash";
+        private readonly string PREVIOUS_PACKAGE_KEY = "previousPackage";
+        private readonly string RELATIVE_BUNDLE_PATH_KEY = "bundlePath";
+        private readonly string STATUS_FILE = "codepush.json";
+        private readonly string UNZIPPED_FOLDER_NAME = "unzipped";
+
+        public CodePushPackage()
+        {
```

Comment:
Do we need to define this? I thought C# auto-gens a parameter less constructor for you.